lib/bloc/inventory/inventory_bloc.dart

```diff
@@ -43,15 +46,32 @@ class InventoryBloc extends Bloc<InventoryEvent, InventoryState> {
             data: event.data,
             count: count,
             enablecomment: event.enablecomment,
-            comment: event.comment);
+            comment: event.comment,
+            enableoverview: event.enableoverview);
         return;
       }
       final inventory = await invapi.inventory();
+      for (var i = 0; i < inventory.userMachine.length; i++) {
```

Comment:
does flutter for foreach or each like this? this kind of coding style is very old now

Reply:
I tried foreach as well but break isn't support for foreach in flutter that's why used it.